api/appendix_extensions.asciidoc

```diff
@@ -25,6 +25,22 @@ alphabetically by author ID.
 Within each group, extensions are listed in alphabetical order by their
 names.
 
+
+[[boilerplate-provisional-header]]
```

Comment:
N.b. this anchor name is hardwired into the generator script and looks a bit odd in context since Vulkan has an entire 'boilerplate' chapter for stuff like this. It would be a bit tedious to change just for the anchor name, but if the wording of the actual generated notice is inappropriate, I can factor that into the API conventions-specific part of the script along with the anchor name.

Reply:
I don't think this is worth worrying about, thanks for the heads up though!